pinot-core/src/main/java/org/apache/pinot/core/common/datablock/RowDataBlock.java

```diff
@@ -48,6 +50,29 @@ public RowDataBlock(ByteBuffer byteBuffer)
     computeBlockObjectConstants();
   }
 
+  @Override
+  public RoaringBitmap getNullRowIds(int colId) {
```

Comment:
This should be part of the `BaseDataBlock`

Reply:
@Jackie-Jiang I mentioned in my other comment that `BaseDataBlock` has two implementations: `RowDataBlock` and `ColumnarDataBlock`. Only `RowDataBlock` has `_rowSizeInBytes` field which we use to calculate position.